src/components/Post/NavBarPost.vue

```diff
@@ -138,6 +138,20 @@ export default {
         this.isLoading = false
         return
       }
+      // check for empty editors
+      let noContent = false
+      for (const editor of Object.values(this.post.currentEditors)) {
+        if (editor.isEmpty) {
+          noContent = true
+          break
+        }
+      }
+      if (noContent) {
+        await this.$bvModal.msgBoxOk('段落內文不得為空白，請輸入段落內文')
```

Comment:
Iteration can be replaced by Array.prototype.every()
```suggestion
      if (Object.values(this.post.currentEditors).every((editor) => !editor.isEmpty))) {
        await this.$bvModal.msgBoxOk('段落內文不得為空白，請輸入段落內文')
      }
```